pkg/entity/project/project.go

```diff
@@ -104,3 +115,8 @@ func manage(args []string, cmd *cobra.Command) {
 	}
 	return
 }
+
+type ProjectsWithActive struct {
```

Comment:
Maybe `ProjectAndScope`? Or something with `Organization`?

Reply:
Organization doesn't cut it as not all projects within an organization can be viewed by just anyone in the organization.
`ProjectsAndScope` is less descriptive to me